Trisha-tech/OnlineBookSales
# Issue#283 - User authentication (Sign up) successful

### Description
A clear and concise description of what the PR does.
- This PR does the following:
  - Fixes user authentication error during Sign up

### Related Issues
Link any related issues using the format `Fixes #issue_number`. 
This helps to automatically close related issues when the PR is merged.
- Placeholder: "Fixes #283 "

### Changes
List the detailed changes made in this PR.
- Fixed a bug in Sign Up page due to multiple sending of tokens after signing up

### Testing Instructions
Detailed instructions on how to test the changes. Include any setup needed and specific test cases.
1. Pull this branch.
2. Run `npm install` to install dependencies.
3. Run `npm test` to execute the test suite.
4. Verify that valid MONGO_URL exists

### Screenshots (if applicable)
![image](https://github.com/user-attachments/assets/1a08891d-7c66-4f79-ba45-8eb1181f8e20)
![image](https://github.com/user-attachments/assets/5834e5ae-66cf-4a15-a301-14ef2bc86763)
![image](https://github.com/user-attachments/assets/9b2bacf0-d948-4751-8f7c-8a0f025f5df8)


### Additional Context
Any additional context or information that reviewers should be aware of.
- After signing up, login in using the same email and password.

### Checklist
Make sure to check off all the items before submitting. Mark with [x] if done.
